src/languages/arcade.js

```diff
@@ -75,7 +249,8 @@ export default function(hljs) {
   ]);
 
   return {
-    name: 'ArcGIS Arcade',
+    aliases: [ 'arcade' ],
```

Comment:
Where did name go?  `alias` isn't necessary if it just matches the filename.

Reply:
not sure, i think this was unintentional. it should be `arcade`, I'll revert `name`.